code/obj/artifacts/artifactdatums.dm

```diff
@@ -3,10 +3,9 @@
 ABSTRACT_TYPE(/datum/artifact/)
 /datum/artifact/
 	var/associated_object = null
-	var/rarity_class = 0
-	// Bigger rarity means its less likely to show up. Thanks for documenting this, guys. - Azungar
-	// Also note that rarity 0 means the artifact does not randomly spawn.
-	// Tweaked rarity 1 to contain all the uninteresting garbage artifacts. Explosion artifacts still appear at 3/4 because explodey is not boring - Phyvo
+	var/rarity = 0
```

Comment:
The name `rarity` sorta implies to me that higher number = rarer and here it's opposite. Not sure what would be a better name. `rarity_weight`?

Reply:
Yeah, I had the same idea, but I didn't want to go for "commonness".^^
Maybe something like occurence_rate or something, but that sounds a bit more complex than necessary.